platform/macos/src/MGLMapView.mm

```diff
@@ -259,7 +259,8 @@ + (NSArray *)restorableStateKeyPaths {
 }
 
 - (void)commonInit {
-    MGLNativeNetworkManager.sharedManager.delegate = MGLNetworkIntegrationManager.sharedManager;
+    [MGLNetworkConfiguration setNativeNetworkManagerDelegateToDefault];
```

Comment:
Incidentally, this is an odd name for a non-setter. `-resetNativeNetworkManagerDelegate` would be more intuitive.

Reply:
I like `reset...`. Have included it in a recent push.